tests/models/test_cpu.py

```diff
@@ -75,57 +70,68 @@ def test_cpu_slurm_save_load(tmpdir):
     # new logger file to get meta
     logger = tutils.get_default_logger(tmpdir, version=version)
 
+    model = BoringModel()
+
+    class _StartCallback(Callback):
+        # set the epoch start hook so we can predict before the model does the full training
+        def on_epoch_start(self, trainer, model):
```

Comment:
```suggestion
        def on_train_epoch_start(self, trainer, model):
```
some changes will be made to `on_epoch_start` in the future so let's change this too.